### ISSUE TITLE
CrossEntropyLoss内置log_softmax，不需要在模型输出层额外添加softmax

### ISSUE THREAD

**Issue by BOB603049648** [illiterate/BertClassifier#2]
问题描述：
- 训练模型时，loss下降非常缓慢
- 数据量大且不同类别数量平衡时能收敛但容易过拟合
- 数据量不大且存在数据不平衡时，模型预测明显偏向于数据量最多的类别

原因分析：

- nn.CrossEntropyLoss()内置log_softmax，模型输出层添加softmax，相当于做了两次的softmax，导致模型输出结果不同类别的权重存在平均化的趋势，模型难以收敛
https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html

解决方案：
- 移除model输出层的softmax
- 若需要获取模型输出的概率值logits，在model输出的logits后接F.softmax()
```
import torch.nn.functional as F

output = model(
                input_ids=input_ids, 
                attention_mask=attention_mask, 
                token_type_ids=token_type_ids, 
            )

logits = F.softmax(output)
```

**Comment by illiterate** [comment]
好的，感谢指出错误，我会尽快修改🤝
